api/app/src/external/commonwell/document/document-download.ts

```diff
@@ -19,6 +20,8 @@ export async function downloadDocument({
   location: string;
   stream: stream.Writable;
 }): Promise<void> {
+  const { log } = Util.out(`CW downloadDocument - M patient ${patientId}`);
```

Comment:
nit: what does the `M` mean in this context? is this a typo?

Reply:
Metriport, so it doesn't eat too much space